src/services/admission/admission-formats.ts

```diff
@@ -0,0 +1,7 @@
+import { DecisionStatus } from "database/decision-db.js";
+
+export interface ApplicantDecisionFormat {
+    _id?: string;
```

Comment:
`_id` is unused elsewhere so it can just be removed. AFAIK none of our types should have `_id`